lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -124,10 +216,17 @@ define([
     return {
         testCourseID: testCourseID,
         testUser: testUser,
+        testCountries: testCountries,
+        testLanguages: testLanguages,
         testTeamDiscussionID: testTeamDiscussionID,
+        createMockTeamData: createMockTeamData,
+        createMockTeams: createMockTeams,
+        createMockTeamMembershipsData: createMockTeamMembershipsData,
+        createMockTeamMemberships: createMockTeamMemberships,
         createMockPostResponse: createMockPostResponse,
         createMockDiscussionResponse: createMockDiscussionResponse,
         createAnnotatedContentInfo: createAnnotatedContentInfo,
-        createMockThreadResponse: createMockThreadResponse
+        createMockThreadResponse: createMockThreadResponse,
+        verifyCards: verifyCards
     };
```

Comment:
As we just discussed, the problem is with the functions that depend upon others. I'm going to leave this alone for this PR, but we can continue the conversation as to a better way to write these helpers.